### ISSUE TITLE
Allow to disable aria-describedby attribute for tooltip component

### ISSUE THREAD

**Issue by dpayk** [chakra-ui/chakra-ui#8042]
### Description

As a developer I want to be able to disable the aria-describedby attribute for a tooltip component in specific cases, to avoid a screenreader from reading duplicate information.

### Problem Statement/Justification

There are cases where aria-label and a tooltip, for example for an icon button, contain the same information. You do not want to remove the aria-label because it should always be available, for example when a screen reader reads a complete section. The tooltip only appears when focussing or hovering the element, so the information would be missing completely when you remove the aria-label for buttons that have a tooltip. At the moment both attributes are there, so that a screenreaders reads the same information twice.

### Proposed Solution or API

Proposed solution is to add a property to the tooltip component that does not set the aria-describedby attribute to the tooltips child element, which is currently set in every case.

**Comment by segunadebayo** [comment]
This use case isn't supported at the moment. Sorry.

You can consider creating a dedicated component that omits the `aria-describedby` property and use that as the tooltip children.
